### ISSUE TITLE
[Windows Containers] Credential manager issue

### ISSUE THREAD

**Issue by dhirschfeld** [AnacondaRecipes/git-feedstock#2]
Even after disabling the credential manager modal dialog:
```powershell
git config --global credential.modalPrompt never
$env:GCM_INTERACTIVE='NEVER'
$env:GCM_PRESERVE_CREDS='TRUE'
```
...cloning a repo still fails from within a container:
```
C:\> git clone https://github.stanwell.com/Trading/skeleton.git
Cloning into 'skeleton'...
bash: Unhandled device /dev/tty

cat: write error: Bad file descriptor
error: failed to execute prompt script (exit code 1)
fatal: could not read Username for 'https://github.stanwell.com': No error
```

Using the actual Windows exe installer this works and prompts for the username and password which are then stored.

Mostly a note to self to investigate further as it would be good if the conda installed git worked well in Windows Containers.

**Comment by dhirschfeld** [comment]
See also https://github.com/conda-forge/git-feedstock/issues/32 for another Windows git/container issue.

**Comment by dhirschfeld** [comment]
Looks like it works with the conda-forge package:
```
C:\> conda install conda-forge::git --force
<snip>
done
C:\> git clone https://github.stanwell.com/Trading/skeleton.git
Cloning into 'skeleton'...
Logon failed, use ctrl+c to cancel basic credential prompt.
Username for 'https://github.stanwell.com': dhirschfeld
Password for 'https://dhirschfeld@github.stanwell.com':
remote: Counting objects: 28, done.
remote: Total 28 (delta 1), reused 1 (delta 1), pack-reused 26
Unpacking objects: 100% (28/28), done.
```

...so it seems fixing one bug introduces another :|

**Comment by dhirschfeld** [comment]
This is fixed in 2.16.3 - git-for-windows/git#1514
